package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "typescript-strict-plugin",
-  "version": "1.1.1",
+  "version": "1.1.0",
```

Comment:
why are we downgrading the plugin version?

Reply:
oops, I think it's because I had an old master on my computer before merging the new one.